swan-lake/integration-tools/asyncapi-tool.md

```
@@ -83,60 +101,86 @@ channels:
+```yaml
+  x-ballerina-auth:
+    header: "X-Webhook-Signature"
+```
+
+  > **Note:** This tag is optional. When absent, no auth code is generated.
+
 ## Usage
 
 After modifying the AsyncAPI contract, the Ballerina sources can be generated using the commands below.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)